src/styles/sidebar/components/menu-item.scss

```diff
@@ -157,3 +157,58 @@ $menu-item-padding: 10px;
 .menu-item__submenu {
   border-bottom: solid 1px var.$grey-2;
 }
+
+// new css below, todo: refactor and cleanup
```

Comment:
CSS is WIP

The css will move up and replace some of the older css above, I wanted to keep all the new rules separate until we're locked into a design.